index.js

```diff
@@ -7,51 +8,48 @@ const STATEBYTES = sodium.crypto_secretstream_xchacha20poly1305_STATEBYTES
 const HEADERBYTES = sodium.crypto_secretstream_xchacha20poly1305_HEADERBYTES
 const KEYBYTES = sodium.crypto_secretstream_xchacha20poly1305_KEYBYTES
 
-if (!TAG_FINAL) throw new Error('JavaScript sodium version needs to support crypto_secretstream_xchacha20poly')
```

Comment:
This was causing a panic in my browser bundle even though the module wasn't actually used, just imported.

Reply:
Can we move this check into the constructor maybe?